llvm/docs/DeveloperPolicy.rst

```diff
@@ -143,54 +143,76 @@ to take on additional community responsibilities beyond code contributions.
 Community members can find active and inactive maintainers for a project in the
 ``Maintainers.rst`` file at the root directory of the individual project.
 
-Maintainers are volunteering to take on the following shared responsibilities
-within an area of a project:
-
-* ensure that commits receive high-quality review, either by the maintainer
-  or by someone else,
-* help to confirm and comment on issues,
-* mediate code review disagreements through collaboration with other
-  maintainers (and other reviewers) to come to a consensus on how best to
-  proceed with disputed changes,
-* actively engage with relevant RFCs,
-* aid release managers with backporting and other release-related
-  activities,
-* be a point of contact for contributors who need help (answering questions
-  on Discord/Discourse or holding office hours).
+Maintainers are volunteering to take on shared responsibilities
+within an area of a project, not to exert power through that status.
+Actions and opinions of maintainers have equal weight to those of other contributors.
+Conflicts are resolved by the area teams (defined in `LP0004
```

Comment:
Reading this before the mediation part makes it sound like all conflict goes to the area teams first.

I think that here you might mean that any conflict that cannot be resolved among contributors, then goes to the area team. The maintainer's opinion is not a tie breaker in that situation, because it holds no more weight than the others'.

Reply:
Although in a social sense, maintainers are maintainers for some reason, so their opinion will likely sway debates in a certain direction regardless. They still have to make a good enough argument though.